book-store_deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata: 
+  name: book-store-deployment
+  namespace: book-store-ns
+  labels:
+    app: books-store
+spec:
+  replicas: 2
+  selector:
+    matchLabels: 
+      app: book-store
+  template:
+    metadata:
+      labels:
+        app: book-store
+    spec:
+      containers:
+      - name: book-store
+        image: book-store
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8000
+---
+apiVersion: v1
+kind: Service
+metadata: 
+  name: book-store-internal-service
+  namespace: book-store-ns
+spec:
+  selector:
+    app: book-store
+  ports:
+    - protocol: TCP
+      port: 8000
```

Comment:
This should be changed to `80` because the ingress is changed to `80` now.
You need to keep the `trargetPort` on `8000` because your `containerPort` is `8000`.

Reply:
look at the my comment on the ingress file